### ISSUE TITLE
How can I fix image uploading problem on redmine 3.3.1

### ISSUE THREAD

**Issue by forianyu** [a-ono/redmine_ckeditor#224]
Can anyone help me with image uploading problem on bitnami_redmine3.3.1?
I rescently uploaded bitnami redmine from 3.1.0 to 3.3.1 and I found that everything is working fine but I can't upload jpg or png images.
It failed when I click <Browse and upload images> menu and when I select an image file.
But I still can attach files using <file> menu.

When I checked the production.log file and c:\windows\temp fold, I found that upload and identify succeeded.
It failed when ruby is trying the run the convert command to create  thumbnail file.

The logs in production.log look like this.
Command :: convert "C:/Windows/Temp/8475ab1a7ddb040822e508512c58bc5e20161020-6752-xruws3.png[0]" -auto-orient -resize "x100" -crop "100x100+14+0" +repage "C:/Windows/Temp/8475ab1a7ddb040822e508512c58bc5e20161020-6752-xruws320161020-6752-1v19ysx"[paperclip] An error was received while processing: #<Paperclip::Error: ["C:/bitnami/redmine-3.3.1-0/ruby/lib/ruby/gems/2.1.0/
gems/cocaine-0.5.8/lib/cocaine/command_line.rb:91:in `run'", "C:/bitnami/redmine-3.3.1-0/ruby/lib/ruby/gems/2.1.0/gems/paperclip-4.2.4/lib/paperclip/helpers.rb:31:in `run'", "C:/bitnami/redmine-3.3.1-0/ruby/lib/ruby/gems/2.1.0/gems/paperclip-4.2.4/lib/paperclip/processor.rb:39:in `convert'", "C:/bitnami/redmine-3.3.1-0/ruby/lib/ruby/gems/2.1.0/gems/paperclip-4.2.4/lib/

Can anyone answer to my question?

My running environments are as follows.
OS: Widows Server 2012 R2 Standard
Redmine: 3.3.1.stable
Ruby: 2.1.9-p490
Rails: 4.2.7.1
Database: Mysql2
CKEditor: 1.1.4
gems: 2.1.0

**Comment by forianyu** [comment]
I solved the problem for myself.

I couldn't upload image from CK editor because convert command failed.
The convert command is used to create thumbnail.

I did the following steps to solve the problem.

1. Add imagemagick directory to PATH
   C:\bitnami\redmine-3.3.1-0\imagemagick
   ※ There is another convert command under c:\windows\system32. So I added the path so that convert under imagemagick is used.

2. Add MAGICK_CODER_MODULE_PATH environment variable.
   C:\bitnami\redmine-3.3.1-0\imagemagick\modules\coders

3. Check if the convert command in the production.log runs without any error.
convert "C:/Windows/Temp/8475ab1a7ddb040822e508512c58bc5e20161020-6752-xruws3.png0" -auto-orient -resize "x100" -crop "100x100+14+0" +repage "C:/Windows/Temp/8475ab1a7ddb040822e508512c58bc5e20161020-6752-xruws320161020-6752-1v19ysx"

4. Reboot the server

**Comment by forianyu** [comment]
I solved the problem for myself.

I couldn't upload image from CK editor because convert command failed.
The convert command is used to create thumbnail.

I did the following steps to solve the problem.

1. Add imagemagick directory to PATH
   C:\bitnami\redmine-3.3.1-0\imagemagick
   ※ There is another convert command under c:\windows\system32. So I added the path so that convert under imagemagick is used.

2. Add MAGICK_CODER_MODULE_PATH environment variable.
   C:\bitnami\redmine-3.3.1-0\imagemagick\modules\coders

3. Check if the convert command in the production.log runs without any error.
convert "C:/Windows/Temp/8475ab1a7ddb040822e508512c58bc5e20161020-6752-xruws3.png0" -auto-orient -resize "x100" -crop "100x100+14+0" +repage "C:/Windows/Temp/8475ab1a7ddb040822e508512c58bc5e20161020-6752-xruws320161020-6752-1v19ysx"

4. Reboot the server
